config/locales/instructions/en.yml

```diff
@@ -67,7 +67,7 @@ en:
         try_again: try again
       sms:
         confirm_code_html: Need another code? %{resend_code_link}. Message rates may apply.
-        number_message_html: We sent a security code to %{number}. This code will expire
+        number_message_html: We sent a one-time code to %{number}. This code will expire
```

Comment:
Same as previous comment, reused in IAL1 (authentication).

https://github.com/18F/identity-idp/blob/34f0bccd457e3ee7fe103f7e073d5fa278e9127c/app/presenters/two_factor_auth_code/phone_delivery_presenter.rb#L11-L17